chapters/2-document-creation-information.md

```diff
@@ -154,7 +154,7 @@ Example:
 Example:
 
     <SpdxDocument rdf:about="http://spdx.org/spdxdocs/spdx-tools-v1.2-3F2504E0-4F89-41D3-9A0C-0305E82...">
-        <rdfs:comment>This document was created using SPDX 2.0 using licenses from the web site.</rdfs:comment>
```

Comment:
The ```rdfs:comment``` is actually correct.   The ```comment``` property is not part of the SPDX namespace, it is part of the [rdfs namespace](https://www.w3.org/TR/rdf-schema/).